features/fixtures/sampler/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,12 @@
       android:icon="@mipmap/ic_launcher"
       android:allowBackup="false"
       android:theme="@style/AppTheme">
+      <meta-data
+        android:name="enable-anr"
+        android:value="@string/MAZE_ENABLE_ANR"/>
```

Comment:
How does that work? There isn't anything in strings.xml and from looking only at the code, I don't really understand where the value is coming from.

Reply:
Ok, I'm using https://github.com/luggit/react-native-config to pull values from a .env into the environment at build time.  They are available in various places, including JS, Gradle, and as strings in the Manifest.  https://github.com/luggit/react-native-config#android